pkgs/applications/editors/notepadqq/default.nix

```diff
@@ -0,0 +1,43 @@
+{ stdenv, fetchFromGitHub, pkgconfig, which, qtbase }:
+
+let
+  version = "v0.53.0";
+  sources = {
+    src = fetchFromGitHub {
+      owner = "notepadqq";
+      repo = "notepadqq";
+      rev = "${version}";
+      sha256 = "0nrngrn69q8jaj1n8i689m7dggqwgsa8fphxixnfmabw5fch2mkk";
+    };
+    codemirror = fetchFromGitHub {
+      owner = "notepadqq";
+      repo = "codemirror";
+      rev = "d790fc39c1a5f06aa66415110b8ebe3026df665a";
+      sha256 = "1xni27gd8fzq0pcdy531hbgr8vazign782dgdy8jm5d3awlsp14a";
+    };
+  };
+in stdenv.mkDerivation {
+  name = "notepadqq-${version}";
+  src = sources.src;
+
+  buildInputs = [
+    pkgconfig which qtbase.qtsvg qtbase.qtwebkit qtbase.qttools
```

Comment:
You can move `pkgconfig` and `which` (if used only during the build) to `nativeBuildInputs`, makes no practical difference outside of cross compilation, though.